src/jaguar.toit

```diff
@@ -51,13 +51,13 @@ main args:
   network := net.open
   socket := network.tcp_listen port
   address := "http://$network.address:$socket.local_address.port"
-  logger.info "running Jaguar device '$name' (id: '$id') on '$address'"
+  logger.info "Running Jaguar device '$name' (id: '$id') on '$address'"
```

Comment:
Please don't change the info strings here.

Reply:
What's the reason for keeping them how they are?
Style, or are they used somewhere else?

Reply:
There are lots of other logs. They all look like 'INFO: running ...'.